pages/index.jsx

```diff
@@ -9,9 +34,21 @@ export default function Home() {
       </Head>
 
       <main>
-        <h1>
-          Hello World!
-        </h1>
+        <h1>Week of April 18 to April 25, 2022</h1>
+
+        <LoadingStatus
+          loadingStatus={loadingStatus}
+        />
+
+        <ul>
+          {loadingStatus === loadingStates.LOADED && teamMembers.map((member, idx) => (
+            <li key={teamMemberIds[idx]}>
```

Comment:
For some more context, here `teamMembers` is being mapped but data from `teamMemberIds` is also being used with the assumption that the data we want has the same index in both arrays. Now we can count on that but if we do any sorting we will not be able to.